puppetboard/utils.py

```diff
@@ -155,8 +158,21 @@ def check_env(env: str, envs: dict):
 
 
 def is_a_test():
-    running_in_shell = any(
-        pytest_binary in sys.argv[0] for pytest_binary in ["pytest", "py.test"]
-    )
+    running_in_shell = any(pytest_binary in sys.argv[0] for pytest_binary in ["pytest", "py.test"])
     running_in_intellij = any("_jb_pytest_runner.py" in arg for arg in sys.argv)
     return running_in_shell or running_in_intellij
+
+
+def dot_lookup(_dict: dict[str, Any], lookup: str) -> str | dict | None:
+    """Recursively look up a value in a dictionary using dot notation string"""
+    lookup_parts = lookup.split(".")
+    if not lookup_parts:
+        return None
+
+    print(f"💕 {_dict=}, {lookup_parts=}")
```

Comment:
Is this some sort of debugging that should have been removed?

Reply:
Yup, sorry about that, removed!